TS7053 in parse-dbf.ts

Hi,

`parseRow` function in `parse-dbf.ts` seems to have a TS error.

```
@loaders.gl/shapefile/src/lib/parsers/parse-dbf.ts(299,5): error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'.
```

A quick & dirty fix could be to add a `@ts-ignore` before `out[field.name] = parseField(text, field.dataType);`

https://github.com/visgl/loaders.gl/blob/master/modules/shapefile/src/lib/parsers/parse-dbf.ts#L299

--- a/modules/shapefile/src/lib/parsers/parse-dbf.ts
+++ b/modules/shapefile/src/lib/parsers/parse-dbf.ts
@@ -290,9 +290,7 @@ function parseRow(
   fields: DBFField[],
   textDecoder: TextDecoder
 ): {[key: string]: any} {
-  const out = {} as {
-		[key: string]: string | number | boolean | null;
-	};
+  const out = {} as {[key: string]: string | number | boolean | null};
   let offset = 0;
   for (const field of fields) {
     const text = textDecoder.decode(

--- a/modules/shapefile/src/lib/parsers/parse-dbf.ts
+++ b/modules/shapefile/src/lib/parsers/parse-dbf.ts
@@ -290,7 +290,7 @@ function parseRow(
   fields: DBFField[],
   textDecoder: TextDecoder
 ): {[key: string]: any} {
-  const out: {[key: string]: string | number | boolean | null} = {}
+  const out: {[key: string]: string | number | boolean | null} = {};
   let offset = 0;
   for (const field of fields) {
     const text = textDecoder.decode(

--- a/modules/shapefile/src/lib/parsers/parse-dbf.ts
+++ b/modules/shapefile/src/lib/parsers/parse-dbf.ts
@@ -296,6 +296,7 @@ function parseRow(
     const text = textDecoder.decode(
       new Uint8Array(view.buffer, view.byteOffset + offset, field.fieldLength)
     );
+    // @ts-ignore
     out[field.name] = parseField(text, field.dataType);
     offset += field.fieldLength;
   }

--- a/modules/shapefile/src/lib/parsers/parse-dbf.ts
+++ b/modules/shapefile/src/lib/parsers/parse-dbf.ts
@@ -290,13 +290,14 @@ function parseRow(
   fields: DBFField[],
   textDecoder: TextDecoder
 ): {[key: string]: any} {
-  const out = {};
+  const out = {} as {
+		[key: string]: string | number | boolean | null;
+	};
   let offset = 0;
   for (const field of fields) {
     const text = textDecoder.decode(
       new Uint8Array(view.buffer, view.byteOffset + offset, field.fieldLength)
     );
-    // @ts-ignore
     out[field.name] = parseField(text, field.dataType);
     offset += field.fieldLength;
   }

--- a/modules/shapefile/src/lib/parsers/parse-dbf.ts
+++ b/modules/shapefile/src/lib/parsers/parse-dbf.ts
@@ -290,7 +290,7 @@ function parseRow(
   fields: DBFField[],
   textDecoder: TextDecoder
 ): {[key: string]: any} {
-  const out = {} as {[key: string]: string | number | boolean | null};
+  const out: {[key: string]: string | number | boolean | null} = {}
   let offset = 0;
   for (const field of fields) {
     const text = textDecoder.decode(

